ephios/core/models/events.py

```diff
@@ -282,7 +284,10 @@ def signup_method(self) -> "BaseSignupMethod":
         try:
             return signup_method_from_slug(self.signup_method_slug, self, event=event)
         except ValueError:
-            return None
+            logger.warning(f"signup method {self.signup_method_slug} was not found")
+            from ephios.core.signup.fallback import FallbackSignupMethod
+
+            return FallbackSignupMethod(self, event=event)
```

Comment:
maybe, to be more semantically coherent, we should return `None` if the slug on the shift is None.

Reply:
Meh it breaks the interface.